### ISSUE TITLE
Send email notification to assignee(s) of a new Publishing Issue

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#4649]
Depends on #4553 

After a new publishing issue was successfully created, an email notification should be sent to selected approver(s).

**From**: email address of a user who created the issue
**To**: email address of an approver
**Subject**: You were assigned to a new issue "``<Issue Title>``" (#``<Issue Id>``)

**Body**:
``<Issue description>``

**[Open Issue button]** (direct link to the "Issue details" dialog) (#4651)

**Issue Items** (``<Total number of items in the issue>``)

Only list items that are stored together with the issue, not references. 

The button should not do anything at this point. Later it will open the Issue Details dialog in the Content Studio. 

![image](https://cloud.githubusercontent.com/assets/11612490/24099200/4d2f4c96-0d6e-11e7-8c6e-bfd0b3819cec.png)
